201-redis-vnet-geo-replication/README.md

```diff
@@ -0,0 +1,12 @@
+# Create two Premium tier Azure Cache for Redis instances with Virtual Networks and Geo-Replication
+
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2Fazure%2Fazure-quickstart-templates%2Fmaster%2F201-redis-vnet-geo-replication%2Fazuredeploy.json" target="_blank">
+    <img src="http://azuredeploy.net/deploybutton.png"/>
+</a>
+<a href="http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F201-redis-vnet-geo-replication%2Fazuredeploy.json" target="_blank">
+    <img src="http://armviz.io/visualizebutton.png"/>
+</a>
+
+Create two Premium tier Azure Cache for Redis instances inside separate Virtual Networks and linked with Geo-Replication by using a template. Virtual Network deployments provide enhanced security and isolation for your Cache instances, as well as isolation using subnets, access control policies, and other features to further restrict access to Azure Redis Cache with a Virtual Network. Geo-replication provide additional availability to application by linking two Premium tier Cache instances and replicating data from the primary cache to the secondary cache.
```

Comment:
Fix the final sentence, I think this is what you want:

Geo-replication provides additional availability to the application by linking two premium tier cache instances and replicating data from the primary cache to the secondary cache.

Reply:
Other than capitalization, is there a difference?